lib/Elevate/Blockers/Base.pm

```diff
@@ -90,7 +91,15 @@ sub has_blocker ( $self, $msg, %others ) {
         $caller_id ||= ref $self;
     }
 
-    my $blocker = cpev::Blocker->new( id => $caller_id, msg => $msg, %others );
+    my $analytics_data;
+    if ( $others{info} ) {
+        my $info = $others{info};
```

Comment:
Not a blocker or anything; but, I believe you could do:
my $info = delete $others{info};
Then you wouldn't need line 99.

Reply:
Good call.  I'll make this change if/when I fix something else up.  Not going to bother if this is the only requested change.